src/multisig/safe.service.ts

```diff
@@ -0,0 +1,37 @@
+import { LoggerService } from '@/lib/logging/logger.service';
+import { Injectable } from '@nestjs/common';
+import SafeApiKit from '@safe-global/api-kit';
+import * as colors from 'colors';
+import { Network } from './multisig.safe.constants';
+
+@Injectable()
+export class SafeService {
+    constructor(private readonly logger: LoggerService) {}
+
+    // function to validate multisig address
+    // if the address is valid the same address is returned
+    // if the address is invalid, the address is reset to empty string
+    async validateMultisigAddress(chainId: number, multisigAddress: string): Promise<void> {
```

Comment:
I suggest creating a new enum `SupportedNetworks` and have `chainId` be of that type.

There are several ways to reuse the `Network` enum for this _(omitting the `Skip` and `Other` values)_. I'll leave the implementation details to you :)